demo/dist/angular-ui-tree.js

```diff
@@ -1860,9 +1860,7 @@
                                                       : (targetElmOffset.top + targetElmOffset.height - childsHeight - ((targetElmOffset.height - childsHeight) * scope.$treeScope.coverage));
 
                       if (elmVertUp <= upLimit) {
-                        if (targetNode.accept(scope, targetNode.index())) {
-                          targetElm[0].parentNode.insertBefore(placeElm[0], targetElm[0]);
-                        }
+                        targetElm[0].parentNode.insertBefore(placeElm[0], targetElm[0]);
```

Comment:
Removing the `accept` check for this statement only ignore the check for placeholder element. So user is allowed to see where he's dropping but if it breaks the `accept` rule the node still can't be dropped.